### PR TITLE
decred: Add sign message tests

### PR DESCRIPTION
**Pull request by matheusd**
Tests signing messages using decred coin.

### CODE REVIEW

File: trezorlib/tests/device_tests/test_msg_signmessage_decred.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        words_nfkd = u"Pr\u030ci\u0301s\u030cerne\u030c z\u030clut\u030couc\u030cky\u0301 ku\u030an\u030c u\u0301pe\u030cl d\u030ca\u0301belske\u0301 o\u0301dy za\u0301ker\u030cny\u0301 uc\u030cen\u030c be\u030cz\u030ci\u0301 pode\u0301l zo\u0301ny u\u0301lu\u030a"
+        words_nfc = u"P\u0159\xed\u0161ern\u011b \u017elu\u0165ou\u010dk\xfd k\u016f\u0148 \xfap\u011bl \u010f\xe1belsk\xe9 \xf3dy z\xe1ke\u0159n\xfd u\u010de\u0148 b\u011b\u017e\xed pod\xe9l z\xf3ny \xfal\u016f"
+
+        sig_nfkd = btc.sign_message(self.client, "Decred", address_n, words_nfkd)
+        assert sig_nfkd.address == "DsbjnfJrnL1orxJBCN8Kf39NjMwEktdfdWy"
+        assert (
+            sig_nfkd.signature.hex()
+            == "1feb2e6eeabe6508fff655c7c3aa7b52098b09e01c9f0cfae404bd4d7baf856e762b72a4c13a8f826b7a42c0b48a5a1b12a96497ca90bd2b183e42f4b3d4eea16b"
+        )
+        print(sig_nfkd)
```
Comment (matejcik) [inline trezorlib/tests/device_tests/test_msg_signmessage_decred.py]: don't print things in tests

File: trezorlib/tests/device_tests/test_msg_signmessage_decred.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+from trezorlib import btc
+
+from .common import TrezorTest
+from trezorlib.tools import parse_path
+
+class TestMsgSignmessageDecred(TrezorTest):
```
Comment (matejcik) [inline trezorlib/tests/device_tests/test_msg_signmessage_decred.py]: please add `@pytest.mark.decred`

**Comment by matejcik** [comment]
is this test currently passing for trezor-core/mcu master? if not, it should be either marked as xfail, or this PR deferred until the relevant code is in the repos

**Comment by matheusd** [comment]
It's passing for mcu master and passing on [trezor-core #358](https://github.com/trezor/trezor-core/pull/358).

I can go either way (mark as xfail or wait until 358 is merged), whatever your preference is.

**Comment by matejcik** [comment]
alright, thanks. let's wait for trezor-core PR

**Comment by prusnak** [comment]
Merged, thanks!

**Pull request merged**